docs/build/reduced.md

```diff
@@ -11,7 +11,7 @@ For applications where package binary size is important, ONNX Runtime provides o
 
 To reduce the compiled binary size of ONNX Runtime, the operator kernels included in the build can be reduced to just the kernels required by your model/s.
 
-For deployment on mobile devices specifically, please read more detailed guidance on [Deploy ONNX Runtime Mobile](../tutorials/mobile/).
+For deployment on mobile devices and web specifically, please read more detailed guidance on [Deploy on mobile device and web](../tutorials/ort-format-model/).
```

Comment:
Do we still want to use the folder `ort-format-model`? Or something like `mobile-web`?

Reply:
as mentioned by @natke we try to minimize the change of paths so let's keep it "ort-format-model" in URL.